pkg/gather/clusterconfig/0_gatherer.go

```diff
@@ -71,6 +71,7 @@ var gatherFunctions = map[string]gathering{
 	"pdbs":                              important(GatherPodDisruptionBudgets),
 	"metrics":                           failable(GatherMostRecentMetrics),
 	"operators":                         important(GatherClusterOperators),
+	"operators_pods_and_events":         important(GatherClusterOperatorPodsAndEvents),
```

Comment:
I think the new one could be "failable" or WDYT? I think it is likely that it will have some errors in case of broken cluster/a lot of degraded clusteroperators.

Reply:
Maybe this assumption is not correct. We should double check.

Reply:
I don't think so... I mean, it wasn't "failable" before when it was together with `operators`. I mean, I agree that once we found a reasonable way to split the logs, definitely the logs should be "failable"... but right now, we are probably fine with the way that it is.